examples/sqlite/getting_started_step_1/src/models.rs

```diff
@@ -1,5 +1,8 @@
 use diesel::prelude::*;
 
+// Order matters here!
+// Queryable assumes that the order of field on the struct matches the columns in the corresponding table.
+// See https://docs.diesel.rs/diesel/deserialize/trait.Queryable.html#derivingfor details
```

Comment:
We should point to the current documentation, instead to the old diesel 1.4 documentation here.
```suggestion
// See https://docs.diesel.rs/master/diesel/deserialize/derive.Queryable.html for details
```